### PR TITLE
add: description how to install krew on fish

### LINKED ISSUE #179: Install example for fish shell
Currently there is only an example for bash and zsh. There should be an example for fish shell as well.

### PR DESCRIPTION
**Pull request by nammn**
Adds a description section to install krew with fish.
This should close #179

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +59,31 @@ Read the [User Guide](./docs/USER_GUIDE.md) for detailed documentation.
 
    and restart your shell.
 
+#### Fish
+1. Make sure that `git` is installed.
```
Comment (ahmetb) [inline README.md]: Please add an empty line before/after lists.

File: README.md [hunk 2]
```diff
@@ -59,6 +59,31 @@ Read the [User Guide](./docs/USER_GUIDE.md) for detailed documentation.
...
+2. Run this command in your terminal to download and install `krew`:
+
+    ```fish
+    begin
+      set -x; cd (mktemp -d) &&
```
Comment (ahmetb) [inline README.md]: What if mktemp returns a path with space in it?
Shouldnt you quote this to prevent splitting?

Comment (nammn) [reply README.md]: Hmm, good question. Actually, I am not that deep into fish, besides using it.
Taking a look.

File: README.md (hunk 1 above)
Comment (nammn) [reply README.md]: 👍

File: README.md (hunk 2 above)
Comment (nammn) [reply README.md]: Just saw this while reading through fish.
https://github.com/fish-shell/fish-shell/issues/159#issuecomment-9533091.
It seems that subcommands in fish prevent space splitting.

**Pull request merged**